data/json/recipes/armor/other.json

```diff
@@ -95,6 +95,30 @@
     "type": "recipe",
     "components": [ [ [ "silver_small", 15 ] ] ]
   },
+  {
+    "result": "bead_bracelet",
+    "category": "CC_ARMOR",
+    "subcategory": "CSC_ARMOR_OTHER",
+    "skill_used": "fabrication",
+    "difficulty": 0,
+    "time": 10000,
+    "reversible": true,
+    "autolearn": true,
+    "type": "recipe",
+    "components": [ [ [ "wooden_bead", 10 ] ], [ [ "string_6", 1 ] ] ]
+  },
+  {
+    "result": "bead_necklace",
+    "category": "CC_ARMOR",
+    "subcategory": "CSC_ARMOR_OTHER",
+    "skill_used": "fabrication",
+    "difficulty": 0,
```

Comment:
```suggestion
```

Reply:
Load error in game when it was set to 0.